src/WordPress/Hooks.php

```diff
@@ -187,8 +187,10 @@ public function getPostRelatedLinks($postId)
 
             foreach ($terms as $term) {
                 $termLink = get_term_link($term);
-                if (!is_wp_error($termLink)) {
+                $termFeedLink = get_term_feed_link($term->term_id, $term->taxonomy);
```

Comment:
@janvitos will this cover both Category feeds or Tag feeds?

Reply:
looking into the docs https://developer.wordpress.org/reference/functions/get_term_feed_link/ it seems cover both.

Reply:
> will this cover both Category feeds and Tag feeds?

Yes it will. But I see you already got your answer :)